tests/Constraints/OfPropertiesTest.php

```diff
@@ -72,22 +72,22 @@ public function getInvalidTests()
                 null,
                 array(
                     array(
-                        "property"   => "prop2",
-                        "pointer"    => "/prop2",
-                        "message"    => "Array value found, but a string is required",
-                        "constraint" => "type",
+                        'property' => 'prop2',
```

Comment:
Is there any option for an alignment rule for this kind of thing? Not a huge problem if there is not, but it would be nice to keep things neat.